src/test/java/org/javarosa/core/model/instance/CsvExternalInstanceTest.java

```diff
@@ -0,0 +1,51 @@
+package org.javarosa.core.model.instance;
+
+import static org.javarosa.test.utils.ResourcePathHelper.r;
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+import org.junit.Before;
+import org.junit.Test;
+
+public class CsvExternalInstanceTest {
```

Comment:
I think having multiple cases in two separate files made this a little hard to follow. I kept having to jump around to see what was really being tested. Not a big deal, but maybe a tweak would be to just have smaller CSVs inline and written to a file that gets parsed in the tests or just smaller more specific CSV test resources? Maybe I'm missing the point, and we need multiple cases in one file.

Reply:
Don't disagree with you. I used the existing tests to get to the coverage I wanted as quickly as possible. I split the window vertically to see the test and resource side by side. If you feel strongly about this, can you please push a commit? I doubt we'll need to look at these tests again any time soon so only if it's really important to you to spend the extra time!

Reply:
I went to spend 10 mins doing this but caught up with some caching problem in IntelliJ so will leave it for the moment. I did a quick rename though: I think part of my problems reading the test were probably due to the `tabSeprated` field, which was the more obvious problem we both missed!